lib/signaling/v2/room.js

```diff
@@ -324,8 +333,51 @@ class RoomV2 extends RoomSignaling {
       this._setupDataTransportBackedNetworkQualityMonitor(roomState.media_signaling.network_quality.transport.label);
     }
 
+    if (!this._trackSwitchOffPromise
+      && roomState.media_signaling
+      && roomState.media_signaling.track_switch_off
+      && roomState.media_signaling.track_switch_off.transport
+      && roomState.media_signaling.track_switch_off.transport.type === 'data-channel') {
+      this._setupTrackSwitchOffMonitor(roomState.media_signaling.track_switch_off.transport.label);
+    }
+
     return this;
   }
+  _teardownTrackSwitchOff() {
+    this._trackSwitchOffPromise = null;
+  }
+
+  _setupTrackSwitchOff(trackSwitchOffSignaling) {
+    trackSwitchOffSignaling.on('updated', (tracksOff, tracksOn) => {
+      this.participants.forEach(participant => {
+        participant.tracks.forEach(track => {
+          if (tracksOff.includes(track.sid)) {
+            track.setSwitchedOff(true /* off */);
```

Comment:
@makarandp0 Nitpick: remove the comment for the function argument?

Reply:
for boolean args I find such inline comments useful to understand what the argument means.